app/views/trainees/employing_schools/_results.html.erb

```diff
@@ -33,4 +33,10 @@
   </div>
 <% end %>
 
+<%= render SchoolResultNotice::View.new(
+  search_query: query, 
+  search_limit: SchoolSearch::DEFAULT_LIMIT, 
```

Comment:
```suggestion
    search_limit: @schools.limit
```

We probably want to reference the limit that was used in the school search service rather than the default, otherwise this could silently break if we start passing a limit in the future.

Reply:
ah actually `@schools` is the list of schools rather than the service itself. I guess it is fine to keep it as is, or we could refactor the controller a bit so that the service itself is kept in a variable.

Reply:
good shout, have made a card https://trello.com/c/6XLafxT9/1505-refactor-schools-search-class-to-return-an-instance-of-itself